openmetadata-ui/src/main/resources/ui/src/components/Ingestion/Ingestion.component.tsx

```diff
@@ -345,6 +384,10 @@ const Ingestion: React.FC<IngestionProps> = ({
       return (
         <Button
           data-testid="deploy"
+          disabled={
+            !isRequiredDetailsAvailable ||
+            !servicePermission?.[ingestion.name]?.EditAll
```

Comment:
We are using this same thing in multiple places, can we optimise it?
```
!servicePermission?.[ingestion.name]?.EditAll
```

Reply:
@Sachin-chaurasiya  updating it!